services/app/app/database.py

```diff
@@ -50,20 +52,11 @@ def player_load(self, username: Optional[str] = None, player_id: Optional[int] =
                 rs = connection.execute("select * from players where players.id = %s", (str(player_id),))
             else:
                 raise Exception('Provide either username or player_id')
-            """
-            if username is not None:
-                inventory = connection.execute("select id from inventory where inventory.playerId = %s", (str(player_id)))
-            else:
-                raise Exception('Inventory could not be loaded. Provide Username')
-          
-            for result in inventory:
-                inv = result['id']
-            """
             for player in rs:
                 return Player(
                     id=player['id'],
                     username=player['username'],
-                    inventory_id="1111", 
+                    inventory_id=random.randint(1000,9999), 
```

Comment:
This has the possibility of overlapping inventory assignments. Maybe, instead of random ints, we can use the autoincrement ID? That is guaranteed to be unique.